packages/gatsby/package.json

```diff
@@ -51,7 +51,7 @@
     "debug": "^3.2.6",
     "del": "^5.1.0",
     "detect-port": "^1.3.0",
-    "devcert-san": "^0.3.3",
+    "devcert": "Js-Brecht/devcert#releases",
```

Comment:
I know that you mentioned this in description, just adding inline comment, that his is kind of a blocker for PR. I could see us potentially using forked version if it would be published on npm (to enable proper version checks) in case it would take long time for upstream to address your PRs there

Reply:
I suppose I _could_ publish the fork.  I'm just reluctant to publish somebody else's work.  That's one reason I started my own project that addresses certificate management... there's several projects out there that do it, but not really any that satisfied my requirements, and I was getting no action when reaching out to the authors.

A couple of PRs I had been watching over at `devcert` finally got merged (took a year), so I hope that this means there's actually going to be some activity over there.

Reply:
> I'm just reluctant to publish somebody else's work.

That's understandable. I'm not going to rush this or anything. I was just saying that we won't be able to merge this PR in current form (it's ok to have this PR open for a bit)

Ignoring being potentially rude by publishing forked version - `devert` is MIT licensed ( https://github.com/davewasmer/devcert/blob/729142dbc826adecc57403356153fca92a042d94/package.json#L23 ), so generally, forking and re-publishing is something that is acceptable. This is the nature of open source, sometimes authors won't have time to maintain their projects, sometimes they will lose interest in doing that, and this is all ok. But it does leave consumers that rely on those packages hanging and only way to unblock is to fork (even if that means to do it temporarily, until things are addressed in main package).

Just keep in mind that if you would end up publishing forked package, it should never be deleted from npm, to not break someone elses dependencies. :)

Reply:
Just reminded me of this story about ["breaking the internet"](https://qz.com/646467/how-one-programmer-broke-the-internet-by-deleting-a-tiny-piece-of-code/) 😆

Reply:
Yup, I was wondering if I should link to left-pad incident, but there was a lot controversial decisions that caused author to unpublish the package :S